src/main/java/com/minecolonies/util/ServerUtils.java

```diff
@@ -79,6 +80,27 @@ public static EntityPlayer getPlayerFromUUID(@NotNull World world, @NotNull UUID
         return players;
     }
 
+    //todo: document

```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)